lib/dice.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+import re
+
+class Dice():
+
+    def split():
```

Comment:
So we can kinda split this up into two separate things. We want a method that the MUD itself can call to get a dice roll, and we want a command that the player can call.  We can accept player input over the command.py and split it up there, then pass the numbers to the roll command.

Reply:
if the MUD wants to roll dice, it should be able to use just the roll function as long as we get it to supply the right arguments, right?